compose/cli/main.py

```diff
@@ -256,16 +257,43 @@ def ps(self, project, options):
         Usage: ps [options] [SERVICE...]
 
         Options:
-            -q    Only display IDs
+            -q          Only display IDs
+            -a, --all   Display orphan containers
         """
+        orphan = options.get('--all', False)
         containers = sorted(
             project.containers(service_names=options['SERVICE'], stopped=True) +
             project.containers(service_names=options['SERVICE'], one_off=True),
             key=attrgetter('name'))
 
+        orphan_containers = sorted(
+            project.containers(stopped=True, orphan=True) +
+            project.containers(one_off=True, orphan=True),
+            key=attrgetter('name'))
```

Comment:
It would be good to avoid these extra API calls when `orphan` is `False`.  How about something like this:

```python

if orphan:
    containers = sorted(
        containers +
        project.containers(...) + 
        project.containers(...),
        key=attrgetter('name'))
```

I think keeping them in the same list will simplify the `-q` case as well.